pkg/controller/controller.go

```diff
@@ -338,9 +338,11 @@ func (c *Controller) handleConfig(config *rbacsyncv1alpha.RBACSyncConfig) error
 
 	for _, binding := range config.Spec.Bindings {
 		// need to valdiate that we only create rolebindings with this configuration.
-		if binding.RoleRef.Kind != "Role" {
+		switch binding.RoleRef.Kind {
+		case "Role", "ClusterRole":
```

Comment:
Can you add a comment here that these are the allowed cases and there is no action to take?  Empty switch statements can feel like bugs without some context.

Reply:
That's a good point. I just removed the comment above the switch and stuck a simple "valid role type" comment in the empty case.

I wonder if I even need this check. I think "Role" and "ClusterRole" are the only two valid role types. I guess it may be paranoid to leave the check in there.

Reply:
Maybe `// RoleRef types valid in a RoleBinding`?

Reply:
👍